src/IIIFPresentation/DLCS/API/DlcsOrchestratorClient.cs

```diff
@@ -0,0 +1,38 @@
+﻿using IIIF.Presentation.V3;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace DLCS.API;
+
+public interface IDlcsOrchestratorClient
+{
+    /// <summary>
+    /// Retrieves a DLCS generated manifest of images for a given presentation manifest id
+    /// </summary>
+    public Task<Manifest?> RetrieveImagesForManifest(int customerId, List<int> batches,
+        CancellationToken cancellationToken = default);
+}
+
+public class DlcsOrchestratorClient(
+    HttpClient httpClient,
+    IOptions<DlcsSettings> dlcsOptions,
+    ILogger<DlcsOrchestratorClient> logger) : IDlcsOrchestratorClient
+{
+    private readonly DlcsSettings settings = dlcsOptions.Value;
+
+    public async Task<Manifest?> RetrieveImagesForManifest(int customerId, List<int> batches,
```

Comment:
Should this be named something else as it's not just retrieving images, it's a manifest. In future the NQ could contain services for Files or AV.